src/app/settings/admin/dynamic-datacenters/datacenter-data-dialog/component.ts

```diff
@@ -51,6 +51,11 @@ enum Title {
   Edit = 'Edit Datacenter',
 }
 
+enum Mode {
```

Comment:
I see that this enum is being declared in multiple places so can we move this to `shared/types` and call it `DialogActionMode` (or something similar)?